com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1790,9 +1790,6 @@ private List<TaskResult> runTasks(IProgress monitor) throws IOException, Compile
 
                 // compare all task signature. current task signature between previous
                 // signature from state on disk
-                TimeProfiler.start("compare signatures");
-                TimeProfiler.addData("color", "#FFC0CB");
-                TimeProfiler.addData("main input", String.valueOf(task.input(0)));
```

Comment:
It's too much data for a big project. And this particular data isn't that useful